### PR TITLE
update composer dependencies

### PR DESCRIPTION
**Pull request by casberkman1**
This pull request updates the depencies for Laravel 6.

Tests still pass.

Could you please have a look when it suits you.

Thanks in advance!

### CODE REVIEW

**Comment by nstapelbroek** [comment]
That's awesome @casberkman1 ! Thanks for the PR. 🎉 

As the release schedule of Laravel changed to semver since 6 I'm not sure if there is an easy way to support both 5 and 6 in the same "branch".

Looking at some of the stuff that spatie does I think it's safe to release this as a new major version and drop the good old PHP versions. I'll add the details as PR feedback and I'd be happy to help on these changes later today.

File: composer.json [hunk 1]
```diff
@@ -18,13 +18,13 @@
         "issues": "https://github.com/nstapelbroek/culpa-laravel-5/issues"
     },
     "require": {
-        "illuminate/support": "~5.1",
-        "illuminate/config": "~5.1",
-        "illuminate/events": "~5.1"
+        "illuminate/support": "~6.0",
```
Comment (nstapelbroek) [inline composer.json]: Since Laravel 6 is using semver now we can use the `^` sign instead of the `~`. Let's replace that in all illuminate dependencies.

**Review by nstapelbroek — CHANGES_REQUESTED** [review]
Alrighty, first off: thanks again 😄 

To get the build green we would have to remove the unsupported PHP versions from Travis. Then we have a good starting point to start releasing. I'd suggest that we:

- Remove PHP versions before 7.2 in the Travis configuration file
- Add PHP 7.4 to the Travis configuration file
- Change the target branch of this PR to develop

**Comment by casberkman1** [comment]
Thanks for your feedback @nstapelbroek!

The changes you requested are edited in the last commit. 

Please take another look at the PR when you find the time. :)

**Review by nstapelbroek — APPROVED** [review]
Awesome!

**Comment by nstapelbroek** [comment]
I'll do an integration test and some finishing touches (readme etc.) before releasing a new version. 

If you cannot wait until it's released, you could target the development branch in your composer.json as a temporary workaround.

**Comment by casberkman1** [comment]
Great, thanks!

**Pull request merged**